_posts/2017/2017-02-23-why-i-love-working-as-a-scrum-master.md

```diff
@@ -0,0 +1,136 @@
+---
```

Comment:
@BirgitBader Serena just told me that she will read the post this afternoon.
Ann-Katrin is in meetings this morning, so I hope to catch her after lunch:)